data-model-for-portal/Microsoft.Compute/virtualMachines.json

```diff
@@ -129,6 +129,81 @@
                 }
             ]
         },
+        {
+            "name": "az vm list",
+            "description": "List details of Virtual Machines.",
+            "path": "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Compute/virtualMachines",
+            "confirmation": false,
+            "help": {
+                "learnMore": {
+                    "url": "https://docs.microsoft.com/cli/azure/vm#az-vm-list"
+                },
+                "parameterSets": [
+                    {
+                        "parameters": [
+                            "[--resource-group]",
+                            "[--show-details]",
+                            "[--vmss]"
+                        ]
+                    }
+                ]
+            },
+            "examples": [
+                {
+                    "description": "List all VMs by resource group.",
+                    "parameters": [
+                        {
+                            "name": "--subscription",
+                            "value": "[path.subscriptionId]"
+                        },
+                        {
+                            "name": "--resource-group",
+                            "value": "[path.resourceGroupName]"
+                        }
+                    ]
+                }
+            ]
+        },
+        {
+            "name": "az vm get-instance-view",
+            "description": "Get instance information about a VM.",
+            "path": "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Compute/virtualMachines/{vmName}?$expand=instanceView",
```

Comment:
"path" should be "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Compute/virtualMachines/{vmName}/instanceView"

Reply:
Thanks for your correction, I have found the url in `env/Lib/site-packages/azure/mgmt/compute/v2022_11_01/operations/_virtual_machines_operations.py`.